node-commons/src/main/java/org/apache/iotdb/commons/udf/utils/UDFDataTypeTransformer.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.commons.udf.utils;
+
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.udf.api.type.Type;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/** Transform Type to TsDataType / TsDataType to Type */
+public class UDFDataTypeTransformer {
```

Comment:
A strange comment :D . May refine like this Transform between {@link org.apache.iotdb.tsfile.file.metadata.enums.TSDataType} and {@link org.apache.iotdb.udf.api.type.Type}